docs/package.json

```diff
@@ -3,19 +3,18 @@
   "version": "32.0.0",
   "private": true,
   "scripts": {
-    "dev": "concurrently -k 'yarn run mdjs' 'node server.js'",
-    "build": "yarn run mdjs-prod && cross-env NODE_ENV=production NODE_OPTIONS=--max-old-space-size=8192 next build",
```

Comment:
intentional to remove NODE_ENV for build and export?

Reply:
Yeah.  [It defaults to `production` for every subcommand except `dev`](https://github.com/zeit/next.js/blob/c6d3484a2112abceebf724fbabb80840ec24e6d1/packages/next/bin/next.ts#L84-L85).